opentelemetry-exporter-geneva/geneva-uploader/Cargo.toml

```diff
@@ -16,8 +16,9 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0", features = ["raw_value"] }
 uuid = { version = "1.0", features = ["v4"] }
 # TODO - support both native-tls and rustls
-reqwest = { version = "0.12", features = ["native-tls", "native-tls-alpn"], default-features = false}
-native-tls = "0.2" 
+# http2 feature is required by hyper-util even when using http1_only()
+reqwest = { version = "0.12", features = ["native-tls", "native-tls-alpn", "http2"], default-features = false}
```

Comment:
`http2` feature is coming from some other indirect dependency, even though we use http1.1 in geneva_uploader